Gemfile.lock

```diff
@@ -412,7 +412,7 @@ DEPENDENCIES
   zeroclipboard-rails
 
 RUBY VERSION
-   ruby 2.3.0p0
+   ruby 2.3.1p112
 
 BUNDLED WITH
-   1.12.5
+   1.13.6
```

Comment:
Ran into an interesting issue here where the `RUBY VERSION` in the master branch was `2.3.0p0`. Pushing to Heroku was rejected due to the Gemfile declaring `2.3.1`. Running bundle resolved, and I believe this file may not have been updated on my original bundle due to having an `.rvmrc` file locally that I ultimately tossed. Subsequent pushes/deploys to cfp-next-dev were successful after I ran bundle again, which updated this file